src/core/export/export.common.js

```diff
@@ -0,0 +1,30 @@
+import {isObject, isArray} from '@grid/core/services/utility';
+
+const flattenObject = (obj, separator = ' ,') => {
+	const result = {};
+	for (let prop in obj) {
```

Comment:
You can use 
```
const props = Object.keys(obj);
props.forEach(prop => {
    // first level
    ...
}
```
instead of
```
for (let prop in obj) {
    // first level
    if (obj.hasOwnProperty(prop)) {
       // second level
        ...
    }
}
```

It'll have the same behavior and can help to reduce deep nesting